### DISCUSSION TITLE
Scroll to top

### DISCUSSION THREAD

**Discussion by mukhrr** [community/community#15124] [Pull Requests]
Please, add a button or something to scroll back to the top on `Pull Requests` and also `Issues` (or anywhere where it takes time to scroll to the top manually).

**Comment by GJordao** [comment]
Just to add to this, when we scroll the PR view the navigation bar gets hidden so it makes hard to switch to another view of the PR.

When testing this I have noticed that if you click the PR name while scrolled it will scroll you back to the top 😁 . Still a floating button would be more clear and user friendly.

<img width="402" alt="image" src="https://user-images.githubusercontent.com/21367039/206211964-57ffb4bb-8db9-49be-84f2-09015a0d7ec0.png">
